src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/unit_tests/OrderlyServerClientTests.kt

```diff
@@ -26,374 +22,27 @@ class OrderlyServerClientTests
     }
 
     private val standardHeaders = mapOf("Accept" to ContentTypes.json)
-    @Test
-    fun `passes through query string to GET`()
```

Comment:
All these tests just moved to `PorcelainAPIClientTests` and test the base class `PorcelainAPIClient` instead. Then just check that `OrderlyServerClient` is an instance of `PorcelainAPIClient`, and that it's configured correctly (instance name and base url).